util/keyfile/Cargo.toml

```
@@ -36,7 +36,7 @@ clap = { version = "3.1", features = ["derive", "env"] }
 displaydoc = "0.2"
 hex = "0.4"
 pem = "1.0"
-prost = "0.10.1"
+prost = "0.10.3"
```

Let's omit the patch version here
```suggestion
prost = "0.10"
```